media/js/plugincheck/lib/plugincheck.js

```diff
@@ -16,9 +16,11 @@
         var type;
         var installedVersion;
         var mimes = navigator.mimeTypes['application/x-java-applet'].enabledPlugin;
+        var properties = Object.getOwnPropertyNames(mimes);
```

Comment:
So this is a bit of a hack in that instead of enumerating over `navigator.MimeTypes` we're instead enumerating over an Array of own properties. This works, but I do wonder if there is a way to avoid enumerating entirely to fix this. I'd be open to suggestions?